server/pbench/bin/dashboard_api_server/create_dashboard.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import sqlite3
```

Comment:
Like we did with sarjitsu, I think we need to support a remote Grafana instance that leverages a shared DB, like postgresql.